test/src/test/java/hudson/model/UpdateCenterTest.java

```diff
@@ -59,7 +63,7 @@ private void doData(String location) throws Exception {
 
         UpdateSite us = new UpdateSite("default", url.toExternalForm());
         UpdateSite.Data data = us.new Data(json);
-        assertTrue(data.core.url.startsWith("http://updates.jenkins-ci.org/"));
+        assertThat(requireNonNull(data.core).url, startsWith("https://updates.jenkins.io/"));
```

Comment:
the issue was very obvious after changing to assertThat rather than assert true

```
java.lang.AssertionError: 
Expected: a string starting with "http://updates.jenkins-ci.org/"
     but: was "https://updates.jenkins.io/download/war/2.253/jenkins.war"
```